com.gitlab.coringao.cavestory-nx.json

```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.gitlab.coringao.cavestory-nx",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cavestory-nx",
+  "finish-args": [
+    "--device=all",
```

Comment:
Isn't this enough?
```
  "--device=dri",
```

Reply:
For graphics alone it would be. It wouldn't for joypads, however.

Reply:
Fair enough.